### DISCUSSION TITLE
Mission mode enemy vessel still called "pirate" mission

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#13742] [Bug reports]
### What happened?

![image](https://github.com/FakeFishGames/Barotrauma/assets/37850646/280f8836-16fb-4970-90ae-37e52115921d)

Pirates don't exist anymore. The mission should probably be renamed to "Enemy vessel" or something such.

### Reproduction steps

1. Start a server, select mission mode
2. Select Pirate mission
3. No pirates

### Version

v1.3.0.2

**Comment by itchyOwl** [comment]
Thanks for reporting. Converted to an issue ticket: #13743
